doc/toolbox-enter.1.md

```diff
@@ -42,19 +42,19 @@ host.
 
 ## EXAMPLES
 
-### Enter a toolbox container using the default image matching the host OS
+### Enter the default toolbox container matching the host OS
 
 ```
 $ toolbox enter
 ```
 
-### Enter a toolbox container using the default image for Fedora 36
+### Enter the default toolbox container for Fedora 36
 
 ```
 $ toolbox enter --distro fedora --release f36
 ```
 
-### Enter a custom toolbox container using a custom image
+### Enter a toolbox container with a custom name
```

Comment:
There's a similar example in `doc/toolbox-run.1.md` that could do with the same treatment.

Reply:
I took the liberty to update the custom container example in `doc/toolbox-run.1.md` too.